module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
I think there would be value in adding a generic `__CHPL_FLAGS__` reference here. In the future, it seems like it would be useful to allow the users to pass additional flags to the compiler. And then you can use that today for the `-M <dir>` flags.